regipy/plugins/system/network_data.py

```diff
@@ -0,0 +1,83 @@
+import datetime
+from regipy.exceptions import RegistryKeyNotFoundException
+
+import logging
+
+from regipy.hive_types import SYSTEM_HIVE_TYPE
+from regipy.plugins.plugin import Plugin
+from regipy.utils import convert_wintime
+
+logger = logging.getLogger(__name__)
+
+INTERFACES_PATH = r'Services\Tcpip\Parameters\Interfaces'
+
+class NetworkDataPlugin(Plugin):
+    NAME = 'network_data'
+    DESCRIPTION = 'Get network data from many interfaces'
+    COMPATIBLE_HIVE = SYSTEM_HIVE_TYPE
+
+    def get_network_info(self, subkey, interfaces=None):
+        if interfaces is None:
+            interfaces = []
+
+        for interface in subkey.iter_subkeys():
+            entries = {
+                "interface_name": interface.name,
+                'last_modified': convert_wintime(interface.header.last_modified, as_json=self.as_json),
+                "dhcp": "enable" if interface.get_value("EnableDHCP") == 1 else "disable",
```

Comment:
I would change the variable name to dhcp_enabled and convert the variable to boolean